### DISCUSSION TITLE
Welcome to svelte-router Discussions!

### DISCUSSION THREAD

**Discussion by jorgegorka** [jorgegorka/svelte-router#93] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions.
  * Share ideas.
  * Anything you find relevant.

**Comment by DhyeyMoliya** [comment]
Is there a working demo Github Repository for svelte-router? It would be very helpful.

**Comment by jorgegorka** [comment]
Hey @DhyeyMoliya   

You can check an example of an app using the router here:  https://github.com/jorgegorka/demanda/tree/master/frontend/src/lib/routes

**Comment by OysterD3** [comment]
Do you need help to convert the library to TS?

**Reply by jorgegorka** [reply]
Hi @OysterD3 

Thank you for the offer but I have no plans of converting the library to TS.

**Comment by si3nloong** [comment]
Thank you @jorgegorka for building this package, but I think we need a major update to solve all the pending issues and introduce some new features.

In my opinion, `ga` shouldn't be a in-scope function, we should drop it and let user to import it manually.

Resolve existing bugs :
- isActive is undefined #57 
- nested route #99 
- anchor trigger full page reload #104 

New features purposed:
- async guard #20 
- introduce another reactive store which can give current path #56 

If you don't have the time to solve these, I would like to help on these

**Reply by jorgegorka** [reply]
Hi @si3nloong 

Your help is much appreciated.

I will focus on solving the bugs and if that's fine with you, you can work on #20 or #56 

Is that ok?

**Reply by si3nloong** [reply]
ok, that's great

**Reply by ghost** [reply]
Yes, thank you @jorgegorka for writing this.

I am actually using the package as a front-end router for my fullstack website templates (I would *probably* use sveltekit, but I don't want to give my clients buggy code as sveltekit is not at v1 yet)!

I've found a few issues while using Svelte with this package. The over-all summary of these issues are on sveltejs/sites#350, but the more concise version is that there are import errors that cause the bundler to not compile vendor components/utilities, resulting in the aforementioned issue.

I hope this help improve the package!

**Edit**: P.S. Feel free to let me know if I need to raise an issue about the subject on this repo.
